addons/payment_stripe/static/src/js/express_checkout_form.js

```diff
@@ -214,4 +238,17 @@ paymentExpressCheckoutForm.include({
             paymentRequest => paymentRequest.update(this._getOrderDetails())
         );
     },
+
+    _getPaymentMethodMapping(code) {
+        switch (code) {
```

Comment:
Why not link?

Reply:
Bc link's code (`link`) in odoo already match the one from Stripe ^^